README.md

```diff
@@ -41,17 +41,17 @@ You can find the latest binaries on our [releases](https://github.com/bandprotoc
 
 ### Building from source
 
-To install BandChain's daemon `bandd`, you need to have [Go](https://golang.org/) (version 1.13.5 or later) and [gcc](https://gcc.gnu.org/) installed on our machine. Navigate to the Golang project [download page](https://golang.org/dl/) and gcc [install page](https://gcc.gnu.org/install/), respectively for install and setup instructions.
+To install BandChain's daemon `bandd`, you need to have [Go](https://golang.org/) (version 1.13.5 or 1.14.x) and [gcc](https://gcc.gnu.org/) installed on our machine. Navigate to the Golang project [download page](https://golang.org/dl/) and gcc [install page](https://gcc.gnu.org/install/), respectively for install and setup instructions.
 
 ## Running a Validator Node on the Bandchain Mainnet
```

Comment:
Can you remove apply mainnet part from now (this repo is under development) I cannot use on current mainnet.